### ISSUE TITLE
[bitnami/postgresql*] Not posible to define archive_command for incremental backup 

### ISSUE THREAD

**Issue by Alvaro-Campesino** [bitnami/containers#51871]
### Name and Version

bitnami/postgresql-repmgr

### What is the problem this feature will solve?

Name and Version
bitnami/postgresql-repmgr

What is the problem this feature will solve?
Right now we have replication but we do not have the ability to define an additional incremental backup in a Volume of choice It will be good either to be able to either:

- provide to the pg_ctl a -c archive_command=' cp % /var/lib/postgresql/backup/${MY_POD_NAME}

- Generate the archive_command value in postgresql.conf based on an environment variable

### What is the feature you are proposing to solve the problem?

An additional variable should be added to the environment of the container

During startup when the file is configured it could be placed in the postgresql.conf

### What alternatives have you considered?

I tried to use configmaps for the postgresql.conf, but it won't work as each node needs to have different primary_conninfo value.
At the same time there is no easy way to add this into the code.

I considered using extraArgs, but those are not args to the pg_ctl, but to the command you override it with.

I am going to try to use a initcontainer or a sidecar to modify the postgresql.conf, but it needs to be done before the postgresql starts so it is doing again all the job already done when the file is created by the image.

**Comment by carrodher** [comment]
Thank you for opening this issue and submitting the associated Pull Request. Our team will review and provide feedback. Once the PR is merged, the issue will automatically close.

Your contribution is greatly appreciated!
